add.js

```diff
@@ -0,0 +1,20 @@
+const sharp = require('sharp');
+const path = require('path');
+
+const prog = path.basename(process.argv[1])
+const args = process.argv.slice(2);
+if (args.length < 2) {
+  console.log(`USAGE: ./${prog} <image> <emoji name>\n`)
```

Comment:
nice.

Reply:
also, what does `prog` stand for?

Reply:
`prog` is just the name of the script, which in this case with `process.argv`, after being basenamed, evaluates to `add.js`.

```
[ '/Users/jon/.nvm/versions/node/v10.16.0/bin/node',
  '/Users/jon/code/readmeio/emojis/add.js',
  '/Users/jon/Desktop/Mask.png.png',
  'owlbert-mask' ]
```